### PR TITLE
Release 3.4.1

### PR DESCRIPTION
**Pull request by parkr**
/cc @jekyll/affinity-team-captains @jekyll/documentation

### CODE REVIEW

**Comment by parkr** [comment]
> This is an excellent description of what happened, and should be required reading for CS students. 🍻

🍻 Thanks, @pathawks! :)

File: docs/_posts/2017-03-02-jekyll-3-4-1-released.markdown [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+months and days.
+
+With this, I encourage all of you to look at your code for *unintended*
+functionality and make a judgement call: if it's allowed, *should it be*?
+If it should be allowed, make it *intended* functionality and test it! I
+know I'll be looking at my code with much greater scrutiny going forward,
+looking for unintended consequences.
+
+Many thanks to our Jekyll affinity team captains who helped out, including
+@pathawks, @pnn, and @DirtyF. Thanks, too, to @ashmaroli for reviewing my
```
Comment (ashmaroli) [inline docs/_posts/2017-03-02-jekyll-3-4-1-released.markdown]: I was pleasantly surprised to see this mention. Thanks for that.
But can this line be changed to a simpler construct:
```diff
- Thanks, too, to @ashmaroli for reviewing..
+ Thanks to @ashmaroli too, for reviewing..
```
Apart from this, its a wonderfully written article. :+1: 
Honored to be a part of a great team.

**Comment by mattr-** [comment]
:+1: :shipit: Great post!

File: docs/_posts/2017-03-02-jekyll-3-4-1-released.markdown (hunk 1 above)
Comment (pnn) [inline docs/_posts/2017-03-02-jekyll-3-4-1-released.markdown]: i find the proposed rewrite harder to understand than the original way somehow? english is weird

**Comment by DirtyF** [comment]
@parkr real honest post and useful learning. I like the humility and wisdom ton here, a good reminder of Jekyll's core values. Kudos to all maintainers and contributors, let's keep this community that way: forgiving and kind.

File: docs/_posts/2017-03-02-jekyll-3-4-1-released.markdown (hunk 1 above)
Comment (parkr) [inline docs/_posts/2017-03-02-jekyll-3-4-1-released.markdown]: Hehe thanks @ashmaroli! I am going to leave it as-is to match my style if you don't mind. I hope you'll take the thanks either way! 😄

Comment (ashmaroli) [inline docs/_posts/2017-03-02-jekyll-3-4-1-released.markdown]: I don't mind at all.. :grin:

**Pull request merged**